NumLib/TimeStepping/TimeInterval.h

```diff
@@ -0,0 +1,41 @@
+/**
+ *
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * File:   TimeInterval.h
+ *
+ * Created on November 26, 2018, 4:44 PM
+ */
+#pragma once
+
+namespace NumLib
```

Comment:
Why is it in NumLib? I would put it in BaseLib.

Reply:
At beginning of my implementation, I had put it in BaseLib. Later on, I thought that this class is related to the time stepping only. Therefore I moved it to  NumLib/TimeStepping. If you still think BaseLib is the best choice for it after reading this reply, I will move it  BaseLib again.

Reply:
Could be BaseLib because it's rather general, or ProcessLib/BC/ because this is the only place where it is used now.

Reply:
Since both of you prefer to move to BaseLib, I have  moved it to there but kept the path of parameter documentation.